### PR TITLE
move to rust 1.49 + dpdk 19.11.6 (LTS)

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Associated with https://github.com/capsule-rs/capsule/pull/120 as well. 

Images have been pushed up to hub.docker. 

- [ ] update submod once https://github.com/capsule-rs/capsule/pull/120 is reviewed/in.

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
there's seems to be some new azure fun in container land, I'll take a gander.

File: docker.mk [hunk 1]
```diff
@@ -111,7 +111,7 @@ run-sandbox:
 
 test-sandbox:
 	@if [ "$$(docker images -q $(SANDBOX))" = "" ]; then \
-	docker pull $(SANDBOX); \
+	docker pull $(SANDBOX_IMG); \
```
Comment (zeeshanlakhani) [inline docker.mk:114]: not needed, fixing.

**Comment by zeeshanlakhani** [comment]
@drunkirishcoder ok. these errors are not our fault!

https://www.ubuntuupdates.org/package/core/bionic/main/proposed/linux-azure-5.4-headers-5.4.0-1032 is a missing package. My guess is the GH-Actions hosts needs to update their kernels as there was a somewhat minor kernel regression. For example, https://packages.ubuntu.com/bionic/linux-headers-5.4.0-1034-azure is more recent and the package is there.

**Comment by zeeshanlakhani** [comment]
So, @drunkirishcoder I've pushed these up locally, and I you can run them and double-test. I think we can just bypass this, as we hope they update the kernel (sub)-version soon.

**Review by drunkirishcoder — APPROVED** [review]
👍

**Comment by zeeshanlakhani** [comment]
Closing and re-opening with 1.50.

**Pull request closed without merging**